.github/workflows/scripts/scanner-fetch-nvd.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
Typically, for larger and 'complex' scripts like this one, you want to set some flags to get partial errors to pop out, such as `set -e` or `set -o pipefail`.